### PR TITLE
Node-based style guide!

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
 - Switch from Sinatra to an Express-based style guide. Also move markup samples into SCSS file comments, since [kss-node](https://github.com/kss-node/kss-node) supports parsing markup from comments, and EJS doesn't support "block" helpers.
 - Remove SCSS-Lint for now, since configuring Bundler was a pain. We'll start using the new hotness shortly.
 - Move Grunt tasks back into `Gruntfile.js`, rather than splitting up in `tasks/` subfolder.

For review: @DoSomething/front-end  :cactus:

### CODE REVIEW

File: styleguide/app.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      version: version,
+      styleguide: styleguide,
+      year: new Date().getFullYear()
+    });
+  });
+
+});
+
+// Start 'er up!
+console.log('Styleguide listening on port 3000.');
```
Comment (sbsmith86) [inline styleguide/app.js]: Is this console.log ok?

Comment (DFurnes) [inline styleguide/app.js]: Yup! Just prints to `stdout` so you get a message through the Grunt task.

**Comment by sbsmith86** [comment]
Looks good to me! I just had one question. Also, just curious, why the move an Express-based style guide?

File: styleguide/app.js (hunk 1 above)
Comment (sbsmith86) [inline styleguide/app.js]: Ah I see! :dancer:

**Comment by DFurnes** [comment]
Just simplifies our stack a bit. It always seemed like the Ruby installation step seemed to trip people up, so hopefully this will make Neue a bit more accessible (for example, for the design team).

**Comment by sbsmith86** [comment]
:+1:

**Pull request merged**